screens/Menu/NearbyInterest.js

```diff
@@ -64,54 +78,332 @@ function NearbyInterest(props) {
+            let jsonResp = await resp.json();
+
+            setjsonElementArray_LOY(getFilteredjsonElementArray_LOY(jsonResp, keyId, MAX_WIDTH, SENSOR));
+
+        } catch (error) {
+
+            alert("An error occured while trying to retrive the information. Please leave this screen and come back again.");
+        }
+    }
+
+    const getFilteredjsonElementArray_LOY = (jsonResp, keyId, MAX_WIDTH, SENSOR, ) => {
+
+        var filtering = jsonResp.results.map(element => {
+
+
+            // to display the web and phone number on the detail page, uncomment this line, lines 140 to 153,  lines 165 to 185, and replace the detail   - C.S.B
+            // fetchDetailData(element.place_id);
+
+
+            return {
+                key: element.name,
+                rating: element.rating,
+                open_hours: element.opening_hours.open_now,
+                address: element.vicinity,
+                latitude: element.geometry.location.lat,
+                longitude: element.geometry.location.lng,
+                reviews: element.user_ratings_total,
+                img: `https://maps.googleapis.com/maps/api/place/photo?maxwidth=${MAX_WIDTH}&photoreference=${element.photos[0].photo_reference}&sensor=${SENSOR}&key=${keyId}`
+            }
+        })
+        return filtering;
+    }
+
+    // const fetchDetailData = async (placeId) => {
+    //     try {
+    //         let keyId = await AsyncStorage.getItem("apiKeyId");
+    //         let resp = await fetch(`https://maps.googleapis.com/maps/api/place/details/json?place_id=${placeId}&fields=formatted_phone_number,website&key=${keyId}`);
+    //         let jsonResp = await resp.json();
+
+    //         setPhone(prevState => [...prevState, jsonResp.result.formatted_phone_number]);
+    //         setWeb(prevState => [...prevState, jsonResp.result.website]);
+
+    //     } catch (error) {
+    //         alert("An error occured while trying to retrive the information. Please leave this screen and come back again.");
+    //     }
+    // }
+
+
+    useEffect(() => {
+        const intervalId = setInterval(() => {
+            getFromScreen();
+        }, 1);
+        fetchData_SGW();
+        fetchData_LOY();
+        return () => clearInterval(intervalId);
+    }, []);
+
+    /**
+     * This map method lets us create an array that will contain the JSON Objects
+     * The reason of this array is because we're taking in two new fields to create the JSON Object
+     * The two new fields are taking the data fetched from the Map Place Detail API
+     * The two fields are the Phone Number and Website 
+     * 
+     * 
+     * However for execissive calls prevention reason we will comment this out, will be used if ever having to demo
+     * You will have to replace the actually arrays in the data={} variable in the flat list, by this array
+     * 
+     * C.S.B
+     */
+
+
+    // if (jsonElementArray_SGW && phone !== undefined && web !== undefined) {
```

Comment:
If you're not planning on uncommenting these lines of code, it is better to simply remove them.

Reply:
I guess, I'll remove them. 😔

Reply:
I feel you bro! Removing line 😢